TSC_Minutes/20241010_TSC_MOM.md

```diff
@@ -0,0 +1,94 @@
+# SEAPATH - TSC Meeting
+
+- Date: 10th October 2024
+- Attendees:
+    - Eloi Bail (Savoir-faire Linux)
+    - Paul Le Guen de Kerneizon (Savoir-faire Linux)
+    - Elinor Montmasson (Savoir-faire Linux)
+    - Erwann Roussy (Savoir-faire Linux)
+    - Jan Hille (Welotec)
+    - Ali Abdulla (???)
+    - Bastien Desbos (Savoir-faire Linux)
+    - Phillippe Cizaire
+    - Yair Podemsky (Red Hat)
+    - L. Morin (Schneider Electric)
+    - J. Dides (Schneider Electric)
+    - David Macdonald (General Electric)
+
+
+# Agenda
+
+- Change of TSC hour (one hour before)
+- Update on branch merges / SEAPATH release
+- SEAPATH on ARM
+- SEAPATH at Enlit Europe
+
+
+## Discussions
+
+- Change of TSC time
+	- 3PM instead of 4PM CET
+	- Will change if no objection
+- Update on branch merges and Seapath release
+	- Presentation by Paul Le Guen de Kerneizon (Savoir-faire Linux)
+		- `20241010_TSC_Seapath_merge_status.pdf`
+	- Effort to align Ceph/Ceph-Ansible on the Yocto branch to the Debian branch
+	- Seapath Yocto Kirkstone
+		- Ceph 16 ported to Yocto kirkstone branch
+		- Problems and limitations encountered:
+			- Ceph 16 is not officially supported on Yocto Kirkstone
+			- Ceph 16 is deprecated
+			- Ansible 2.10 is required for Ceph-ansible of Ceph 16, and Yocto kirkstone has use Python 3.10.5 that is not compatible with Ansible 2.10
+				- --> Backports and patches had to be done, error prone
+		- Use of these deprecated tools should be temporary
+	- Seapath Yocto Scarthgap
+		- Tried to use Ceph 16 to keep versions aligned
+			- Note: Ceph 18 could be use with Debian 12 with a backport, but not officially
```

Comment:
remove the "but not officially"
Ceph 18 is officially provided by the ceph team for debian 12. But not by the debian team directly